insonmnia/node/common.go

```diff
@@ -0,0 +1,18 @@
+package node
+
+import "google.golang.org/grpc"
+
+// credentials describe any type of auth data.
+// Empty for now, but must be implemented in nearest future
+type credentials interface{}
+
+// initGrpcClient creates new client connection on addr and wraps it with credentials
+func initGrpcClient(addr string, _ credentials) (*grpc.ClientConn, error) {
+	cc, err := grpc.Dial(addr, grpc.WithInsecure(),
```

Comment:
Another copy of this code. How about making a library method for connecting with sonm specific parameters?